modules/age.nix

```diff
@@ -61,34 +61,45 @@ with lib; let
     }
   '';
 
-  installSecret = secretType: ''
-    ${setTruePath secretType}
-    echo "decrypting '${secretType.file}' to '$_truePath'..."
-    TMP_FILE="$_truePath.tmp"
+  # This function is called with the following mapping:
+  # secretType.symlink = $1
+  # secretType.name = $2
+  # secretType.path = $3
+  # secretType.file = $4
+  # secretType.mode = $5
```

Comment:
I'd rather put this into code like:

```
SYMLINK=$1
NAME=$2
PATH=$3
FILE=$4
MODE=$5
````
so that people don't have to look back to this mapping while reading the function.

Reply:
Makes sense! Changed.